Core/AppRuntime/Source/AppRuntime_Android.cpp

```diff
@@ -14,6 +14,16 @@ namespace Babylon
     {
         std::stringstream ss{};
         ss << "Uncaught Error: " << error.what() << std::endl;
+
+        try
+        {
+            throw;
```

Comment:
This is assuming we are being called in a catch and ignoring the argument. Are we not allowed to do this?
```suggestion
            throw error;
```

Reply:
`throw error;` crashes the app.

Reply:
I think we have to capture the exception_ptr instead and pass that along.